includes/activity/class-postal-address.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * PostalAddress is a custom ActivityPub object firstly used by Mobilizon
+ * derived from https://schema.org/PostalAddress.
+ *
+ * @license AGPL-3.0-or-later
+ */
+
+namespace Activitypub\Activity;
```

Comment:
Should we use `namespace Activitypub\Activity\Object;` (and move them into a subfolder) instead, to differentiate between core concepts (Object, Actor, Activity) and sub-types?

Reply:
> The first confusion for me was the namespace starting with Activitypub\Activity\ as 'Activity' is already a subtype.

I see it from that perspective: https://www.w3.org/TR/activitystreams-vocabulary/#types

There are core types, like Object, Link, Activity, IntransitiveActivity, Collection, ... and Extended Types: https://www.w3.org/TR/activitystreams-vocabulary/#extendedtypes

So that's why I thought it might be enough to stick with the core types, to keep it simple... but I agree that a sub-structure helps a lot with complicated formats like events, but let's keep them as sub-types and maybe put them in subfolders/namespaces then.

Reply:
So you are right, but not with `Activity`, but with `Actor` beeing an extended type, but it makes still sense to have them on top level ideas (at least in my head 🫣 )

Reply:
Kind of

```
activity/
├─ object/
│  ├─ class-event.php
│  ├─ class-place.php
│  ├─ ...
├─ class-base-object.php
├─ class-actor.php
├─ class-activity.php

```

...and if it makes sense to also have `Activity` subtypes, put them in the sub-directory and the same with `Actors`.

Reply:
Or maybe that makes more sense: The objects at the main level are all the main "idea" of an Acticity/Object/Actor and you have to make it more concrete using the `type` attribute. For `Actor` these are `Person`, `Application`, ... and for `Object` it is `Event`, `Note`, ... and for `Activity` it is `Create`, `Update`, ...